examples/visualization/plot_clickable_image.py

```diff
@@ -24,10 +25,12 @@
 
 # Set parameters and paths
 plt.rcParams['image.cmap'] = 'gray'
-im_path = op.join(op.dirname(mne.__file__), 'data', 'image', 'mni_brain.gif')
+
+im_path = op.join(op.dirname(__file__), '..', '..', 'mne', 'data', 'image',
+                  'mni_brain.gif')
 
 # We've already clicked and exported
-layout_path = op.join(op.dirname(mne.__file__), 'data', 'image')
+layout_path = op.join(op.dirname(__file__), '..', '..', 'mne', 'data', 'image')
```

Comment:
What's the failure? Is there some way to use the location of `mne` instead of the location of the example file? It really should be more robust to use the repo-location to find data files from the repo than use the example-location -- people could reasonably just download the example to their Desktop and expect it to work. Maybe we need to add `mne.data_path` within the repo that just points to `op.join(os.abspath(__file__), 'data')` or so in `__init__.py`.